src/popup/components/__snapshots__/no-tickets.test.tsx.snap

```diff
@@ -18,13 +18,9 @@ exports[`no-tickets with errors renders an error report 1`] = `
       <br />
       while scanning for tickets.
     </p>
-    <p />
```

Comment:
This is a "self-closing" `p`-tag, it should be equivalent to `<p></p>`. While I can't tell where these are from based on the diff here, it doesn't look like the `ul` was nested inside a `p` like you mention in the PR description. 🤔

Reply:
For reference: https://reactjs.org/docs/jsx-in-depth.html

Reply:
Since it's only a mocked case it doesn't matter much. I just noticed and was wondering what error message you saw. 🙂

Reply:
Ah. That makes sense. They're wrapped one level above. 👍